src/arviz_plots/plots/distplot.py

```diff
@@ -281,7 +287,12 @@ def plot_dist(
             )
 
             plot_collection.map(
-                hist, "hist", data=density, ignore_aes=density_ignore, **density_kwargs
+                hist,
+                "hist",
+                data=density,
+                ignore_aes=density_ignore,
+                step_hist=step_hist,
+                **density_kwargs,
```

Comment:
```suggestion
                **density_kwargs,
```

step_hist defaults to "False" if the user wants a step histogram, then it should passed to `density`